streams/src/main/java/org/apache/kafka/streams/state/internals/ValueAndTimestampSerializer.java

```diff
@@ -34,6 +34,25 @@
         timestampSerializer = new LongSerializer();
     }
 
+    public static boolean maskTimestampAndCompareValues(final byte[] left, final byte[] right) {
```

Comment:
req: Please add unit tests for this method.

Reply:
Yep, got this done.